CIPs/CIP-8/CIP-8.md

```diff
@@ -12,113 +12,112 @@ requires: Doctypes (CIP-5)
 
-**Signed records** of a Tile document are encoded using [dag-jose](https://github.com/ceramicnetwork/js-dag-jose). Other than that they conform to the standard record format. The content of the `data` property should be a [json-patch](http://jsonpatch.com/) object which contains the difference from the previous document content state.
+**Signed commits** of a Tile document are encoded using [dag-jose](https://github.com/ceramicnetwork/js-dag-jose). Other than that they conform to the standard commit format. The content of the `data` property should be a [json-patch](http://jsonpatch.com/) object which contains the difference from the previous document content state.
 
 ### State transitions
 
-Below the state transition logic for the three different supported records is described.
+Below the state transition logic for the three different supported commits is described.
 
-#### Applying the Genesis Record
+#### Applying the Genesis Commit
 
-When the genesis record is applied a `DocState` object is created that looks like this:
+When the genesis commit is applied a `DocState` object is created that looks like this:
 
 ```js
 {
   doctype: 'tile',
   metadata: {
-    owners: genesisRecord.owners,
-    schema: genesisRecord.schema,
-    family: genesisRecord.family,
-    tags: genesisRecord.tags
+    owners: genesisCommit.owners,
+    schema: genesisCommit.schema,
+    family: genesisCommit.family,
+    tags: genesisCommit.tags
   },
-  content: genesisRecord.data,
+  content: genesisCommit.data,
   signature: SignatureStatus.SIGNED,
   anchorStatus: AnchorStatus.NOT_REQUESTED,
-  log: [new CID(genesisRecord)]
+  log: [new CID(genesisCommit)]
 }
 ```
 
 In addition to this, the following should be verified:
 
-* If the record uses the *dag-cbor* format (not signed), `genesisRecord.data` MUST equal `null`
+* If the commit uses the *dag-cbor* format (not signed), `genesisCommit.data` MUST equal `null`
 * `owners` is an array of DID strings
-* If `schema` is defined it should be Ceramic DocId string
+* If `schema` is defined it should be Ceramic StreamId string
 * If `tags` is defined it should be an array of strings
 * `content` should be JSON
 * If `schema` is defined the data in the `content` should be valid according to the schema
 * The **dag-jose** signature should be valid and signed by the DID(s) in the `owners` array
 
 
-#### Applying a Signed Record
+#### Applying a Signed Commit
 
-When a signed record is applied the `DocState` object should be modified in the following way:
+When a signed commit is applied the `DocState` object should be modified in the following way:
 
 ```js
 state.next = {
-  content: applyJSONPatch(state.content, signedRecord.data),
+  content: applyJSONPatch(state.content, signedCommit.data),
   metadata: {
-    schema: signedRecord.schema,
-    family: genesisRecord.family,
-    tags: signedRecord.tags
+    schema: signedCommit.schema,
+    family: genesisCommit.family,
```

Comment:
is this trying to imply that `family` can't be changed?  Because we do actually allow changing `family` in the new API

Reply:
I guess. Don't want to get into a discussion about what the correct behavior should be. Can we keep focus on the main things this PR is changing, and do other changes separately / later.